content/coordination/onboarding/index.en.md

```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
 
         -   Country
 
-        -   Organisers
+        -   Organizers
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.

Reply:
```suggestion
        -   Organisers
```